shell/plugins/clean-html-directive.js

```diff
@@ -17,6 +17,13 @@ const ALLOWED_TAGS = [
   'strong',
 ];
 
+DOMPurify.addHook('afterSanitizeAttributes', (node) => {
```

Comment:
@aalves08 yeah - Not so keen on this solution.

I assume all of our links that open in a new tab or broken? If so, I'd suggest removing this form this PR and filing an issue for that bug with the clean-html directive and that we fix it there and get this PR in.

Feels like we need a way to allow the target to be allowed by the clean directive, rather than adding it when we see the other attributes.

Reply:
Ref might be useful: https://github.com/cure53/DOMPurify/issues/317

Reply:
Changes reverted. Will create another issue and PR.